patcher/src/buttercup/patcher/agents/leader.py

```diff
@@ -36,7 +36,7 @@ class PatcherLeaderAgent(PatcherAgentBase):
     tasks_storage: Path
     model_name: str | None = None
 
-    def _init_patch_team(self) -> StateGraph[PatcherAgentState, PatcherAgentState, PatcherAgentState]:
+    def _init_patch_team(self) -> StateGraph[PatcherAgentState, PatcherConfig, PatcherAgentState, PatcherAgentState]:
```

Comment:
Was PatcherConfig just missing before?

Reply:
no, I think it's due to new langgraph version